packages/tracing/test/hub.test.ts

```diff
@@ -183,6 +183,14 @@ describe('Hub', () => {
         expect(transaction.sampled).toBe(false);
       });
 
+      it('should not try to override sampling decision provided in transaction context', () => {
+        // setting tracesSampleRate to 1 means that without the override, the sampling decision should be true
+        const hub = new Hub(new BrowserClient({ tracesSampleRate: 1 }));
+        const transaction = hub.startTransaction({ name: 'dogpark', sampled: false });
+
+        expect(transaction.sampled).toBe(false);
```

Comment:
[M]

This is good, though seems incomplete given that there are more ways to influence the sampling decision now.
I'd expect to see a permutation of tracesSampleRate 0/1, tracesSampler forcing true/false and starting a transaction with sampled: true/false/undefined. Perhaps some of those cases are covered by other tests, I doubt all?!

Reply:
Okay - I think I got all the permutations. It did up the branch coverage!